src/supermarket/config/routes.rb

```diff
@@ -20,6 +20,7 @@
       delete "cookbooks/:cookbook" => "cookbook_uploads#destroy"
       delete "cookbooks/:cookbook/versions/:version" => "cookbook_uploads#destroy_version", constraints: { version: VERSION_PATTERN }
       get "users/:user" => "users#show", as: :user
+      get "universe" => "universe#index"
```

Comment:
Can we add a redirect instead of 2 different routes 
https://stackoverflow.com/questions/3622706/creating-a-rails-route-to-an-external-url